README.md

```diff
@@ -2,9 +2,13 @@
 
 @inrupt/solid-client is a JavaScript library for accessing data and managing permissions on data stored in Solid Pods. It provides an abstraction layer on top of both Solid and Resource Description Framework (RDF) principles and is compatible with the RDF/JS specification. You can use solid-client in Node.js using CommonJS modules and in the browser with a bundler like Webpack, Rollup, or Parcel.
 
+@inrupt/solid-client is part of a family open source JavaScript libraries designed to support developers building Solid applications.
+
 # Inrupt Solid JavaScript Client Libraries
 
-Inrupt Solid JavaScript Client Libraries are a group of libraries designed to support developers building Solid applications.
+## Data access and permissions management - solid-Client
+
+[@inrupt/solid-client](https://github.com/inrupt/solid-client-js) allows developers to access data and manage permissions on data stored in Solid Pods.
```

Comment:
I don't see the value of redirecting people to the page they are currently browsing.

Reply:
To be fair, you can view the README in different locations, e.g. on npmjs.com, in your local checkout, in your node_modules, ...

That said, I would link to the npmjs.com page instead of the source code :)

Reply:
Good point, I didn't realize the README would be bundled and shared in different places. I still think it's a bit confusing to introduce the current library in a list of related third-party libraries, but I don't have a strong opinion on this, so it it's valuable to have the same README snippet accross the different libraries, happy to leave this in.

Reply:
As Nicolas pointed out, the goal of keep users informed of the suite of libraries by keeping the same snippet across all README files.